event.go

```diff
@@ -185,6 +188,18 @@ func Dict() *Event {
 	return newEvent(nil, 0)
 }
 
+// GetContext allows retrieval of the context.Context which is optionally
+// stored in the event.  This allows Hooks to retrieve values which are stored
+// in the context.Context.  This can be useful in tracing, where span
+// information is commonly propagated in the context.Context.
+// See also Logger.InfoCtx, etc.
+func (e *Event) GetContext() context.Context {
```

Comment:
`Context` instead of `GetContext` would be more idiomatic.

Reply:
Good point!